File: QTD2.Domain/Entities/Core/Version_Task_Tool_Link.cs
namespace QTD2.Domain.Entities.Core
{
    public class Version_Task_Tool_Link : Common.Entity
    {
        public int Version_TaskId { get; set; }

        public int Version_ToolId { get; set; }

        public virtual Version_Task Version_Task { get; set; }

        public virtual Version_Tool Version_Tool { get; set; }

        public Version_Task_Tool_Link()
        {
        }

        public Version_Task_Tool_Link(int version_TaskId, int version_ToolId)
        {
            Version_TaskId = version_TaskId;
            Version_ToolId = version_ToolId;
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/Version_Task_Tool_Link/Version_Task_Tool_Link.ts
import { Entity } from '../Entity';
import { Version_Task } from '../Version_Task/Version_Task';
import { Version_Tool } from '../Version_Tool/Version_Tool';

export class Version_Task_Tool_Link extends Entity {
  version_TaskId!: any;
  version_ToolId!: any;
  version_Task!: Version_Task;
  version_Tool!: Version_Tool;
}
